### PR TITLE
suanpan: Add version 1.5

### PR DESCRIPTION
**Pull request by TLCFEM**

### CODE REVIEW

File: bucket/suanpan.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "notes": [
+        "Please ensure Microsoft Visual C++ Redistributable for Visual Studio 2019 is installed. You can install it manually or via scoop.",
+        "You can also find and run bundled AddAssociation.bat to configure file association and Sublime Text syntax support."
+    ],
+    "suggest": {
+        "vcredist": ["vcredist2019-portable", "vcredist2019"]
+    },
+    "checkver": {
+        "url": "https://github.com/TLCFEM/suanPan/releases",
+        "regex": "suanPan-v([0-9]\\.[0-9]\\.?[0-9]?)"
```
Comment (Ash258) [inline bucket/suanpan.json:18]:
```suggestion
        "github": "https://github.com/TLCFEM/suanPan",
        "regex": "download/suanPan-v([\\d.]+)/"
```

File: bucket/suanpan.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "homepage": "https://tlcfem.github.io/suanPan/",
+    "description": "An Open Source, Parallel and Heterogeneous Finite Element Analysis Framework",
+    "license": "GPL-3.0-or-later",
+    "notes": [
+        "Please ensure Microsoft Visual C++ Redistributable for Visual Studio 2019 is installed. You can install it manually or via scoop.",
+        "You can also find and run bundled AddAssociation.bat to configure file association and Sublime Text syntax support."
+    ],
+    "suggest": {
+        "vcredist": ["vcredist2019-portable", "vcredist2019"]
+    },
```
Comment (Ash258) [inline bucket/suanpan.json:15]:
```suggestion
    "version": "1.5",
    "homepage": "https://tlcfem.github.io/suanPan/",
    "description": "Parallel and heterogeneous finite element analysis framework",
    "license": "GPL-3.0-only",
    "notes": "You can run \"$dir\\AddAssociation.bat\" to configure file association and Sublime Text syntax support.",
    "suggest": {
        "vcredist2019": "vcredist2019"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/TLCFEM/suanPan/releases/download/suanPan-v1.5/suanPan-win-mkl-vtk.zip",
            "hash": "de9f1f1ed3d3d9c92e5c3c875cd22c028ea73b4426c08cc5797f4417a8d16cb5"
        }
    },
    "bin": "suanPan.exe",
```

File: bucket/suanpan.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "checkver": {
+        "url": "https://github.com/TLCFEM/suanPan/releases",
+        "regex": "suanPan-v([0-9]\\.[0-9]\\.?[0-9]?)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/TLCFEM/suanPan/releases/download/suanPan-v$version/suanPan-win-mkl-vtk.zip",
+        "hash": {
+            "url": "https://github.com/TLCFEM/suanPan/releases/download/suanPan-v$version/suanPan-win-mkl-vtk.sha256"
```
Comment (Ash258) [inline bucket/suanpan.json:23]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/TLCFEM/suanPan/releases/download/suanPan-v$version/suanPan-win-mkl-vtk.zip"
            }
        },
        "hash": {
            "url": "$baseurl/suanPan-win-mkl-vtk.sha256"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — CHANGES_REQUESTED** [review]
Does not work

image

**Comment by TLCFEM** [comment]
Given that the binary is statically linked, I doubt if that is the problem of vcredist. Do you mind to check if vcredist is the latest version, in case it is not?

**Comment by TLCFEM** [comment]
I have tested the package with Win10 virtual environment provided by MS [https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/).

The program can be successfully executed after confirming the "Unknown Publisher" warning.
powershell_PLnhJJSRXP

It can be executed via scoop as well.
powershell_Jw9fzHxp2P

@Ash258 Please see what to do next.

**Comment by TLCFEM** [comment]
@Ash258 Another thing is, the program requires AVX.

**Comment by issaclin32** [comment]
Already exists in the main bucket (https://github.com/ScoopInstaller/Main/pull/2754)

**Pull request closed without merging**